geopolars/src/geoseries.rs

```diff
@@ -96,6 +116,13 @@ pub trait GeoSeries {
     /// https://docs.rs/geo/latest/geo/algorithm/simplify/trait.Simplify.html for details
     fn simplify(&self, tolerance: f64) -> Result<Series>;
 
+    /// Returns a GeoSeries with each of the geometries skewd by a fixed x and y amount around a
+    /// given origin
+    fn skew(&self, xs: f64, ys: f64, origin: TransformOrigin) -> Result<Series>;
```

Comment:
Is this the same operation as geopandas' [`GeoSeries.skew`](https://geopandas.org/en/stable/docs/user_guide/geometric_manipulations.html#GeoSeries.skew)? Just wondering because their api takes in an `angle` as argument.

Reply:
The angle is expressed along each axis with the xs and ys directions. Have added some docs to clarify